src/General/Typography/Title.test.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import Title from './Title';
+import { PrimaryColor } from '../../Style/Colors';
+
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+
+const titleText = 'Glints Aries';
+
+describe('displays the correct text and uses the correct html tag for each tag:', () => {
+  const tags: any[] = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'];
```

Comment:
Can use the following to instead of `any[]`
``` 
 type tagsType = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
 const tags: tagsType[] = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'];
```